frontend/client-list/table/desktop-clients-table.component.tsx

```diff
@@ -92,7 +92,8 @@ export default function DesktopClientsTable(props: ClientsTableProps) {
                   name="client-checked"
                   aria-label={`Select ${client.fullName}`}
                   value={client.id}
-                  onClick={evt => evt.stopPropagation()}
+                  checked={props.isSelected[client.id]}
```

Comment:
This is the problem child line for the second error. 

Using a ternary, a separate function that returned true or false, and having the unclick change the value from true to false did not work. This way works on the surface, but still kicks out a typescript error.